app/views/pregnancies/_call_log.html.erb

```diff
@@ -3,14 +3,29 @@
 		<h1 class="border-bottom title"> Call Log</h1>
 		<table class="table border-side">
 		<tr>
-			<th>Time Stamp</th>
+			<th>Date</th>
+			<th>Time</th>
 			<th>Result</th>
 			<th>user</th>
 		</tr>
+		<div class="btn btn-primary" id="toggle-call-log">View all calls</div>
 		<tbody>
-			<td><%= pregnancy.initial_call_date %></td>
-			<td><%= pregnancy.status %></td>
-			<td><%= pregnancy.user %></td>
+			<% @pregnancy.recent_calls.each do |call| %>
```

Comment:
Minor change, but for consistency -- `pregnancy` should be available within the partial as a local variable -- any chance I could get you to use the local variable instance of the instance variable here?